lib/course_planner/tasks.ex

```diff
@@ -2,17 +2,17 @@ defmodule CoursePlanner.Tasks do
   @moduledoc false
   alias CoursePlanner.Repo
   alias CoursePlanner.Tasks.Task
-  alias Ecto.{DateTime, Changeset}
+  alias Ecto.Changeset
   import Ecto.Query, except: [update: 2]
 
-  @tasks from t in Task, where: is_nil(t.deleted_at), preload: [:user]
-
   def all do
-    Repo.all(@tasks)
+    Task
+    |> Repo.all()
+    |> Repo.preload(:user)
```

Comment:
@ghatighorias I mean outside of `all/0` function when you call it or define a different one like `all_with_users/0` or `all(resource), do: Repo.all() |> Repo.preload(resource)`.